app/src/main/java/com/example/anfcodetest/MainAdapter.kt

```diff
@@ -9,27 +9,15 @@ import androidx.recyclerview.widget.RecyclerView
  * Created by Matt Kleman 09/04/20
  */
 
-class MainAdapter(val cardList: List<Card>): RecyclerView.Adapter<CustomViewHolder>() {
+class MainAdapter(private val cardList: List<Card>): RecyclerView.Adapter<CustomViewHolder>() {
 
-    override fun getItemCount(): Int {
```

Comment:
Can we make this an expression (inline) function and remove the explicit return type, to make it a little more readable and save us a few lines? [Expression functions](https://kotlinlang.org/docs/reference/functions.html#single-expression-functions)